### PR TITLE
DOCSP 32921 Add warning for --loadLevel setting

### PR DESCRIPTION
**Pull request by ltran-mdb2**
## DESCRIPTION

Add warning for loadLevel setting

## STAGING

https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-32921/reference/mongosync/#std-option-mongosync.--loadLevel
https://docs-mongodbcom-staging.corp.mongodb.com/cluster-sync/docsworker-xlarge/DOCSP-32921/reference/configuration/#mongodb-setting-loadLevel

## JIRA

https://jira.mongodb.org/browse/DOCSP-32921

## BUILD LOG

https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6500baf224fcc731b4809d7f

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/reference/configuration.txt [hunk 1]
```diff
@@ -100,6 +100,11 @@ Options
    To set the ``loadLevel`` setting from the command line, see the
    :option:`--loadLevel` option.
 
+   .. warning::
+
+      Setting the ``loadLevel`` higher than the default may negatively impact 
+      the performance of the destination cluster. 
+
```
Comment (sarah-olson-mongodb) [inline source/reference/configuration.txt:107]: (minimalism)
Consider the following minor tweaks to make the sentence a bit simpler. Something like:
"Setting ``loadLevel`` higher than the default may negatively impact the destination cluster performance." 

(question/level of detail) 
Does it make sense contextually to say what the default is?

**Review by sarah-olson-mongodb — COMMENTED** [review]
Thanks a ton @ltran-mdb2! This looks great. I made a few optional suggestions. In addition, because you are using the same text in two locations, what do you think about using an include instead?

File: source/reference/configuration.txt (hunk 1 above)
Comment (ltran-mdb2) [reply source/reference/configuration.txt:104]: Thanks for the feedback @sarah-olson-mongodb ! I changed the warning to an includes and took your suggestions! I think it does make sense to specify what the default load level is.

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @ltran-mdb2! LGTM

**Pull request merged**